serveradmin/apps/migrations/0004_application_last_login.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2019-07-18 11:52
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('apps', '0003_public_key_length'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
What is the reason we do not store every single login but "just" the last one?

Reply:
See: https://github.com/innogames/serveradmin/pull/71/files#diff-e62344281bdb7c2545658a551867dc62R142

We don't have sessions so each request is authenticated. Some apps like puppet send thousands of request per second. Each of them would cause an additional database commit and hence an additional flush to disk. That's pretty expensive. We only care about which apps we can disable here anyway. If you want that information you can look into the logs / send them to kibana.

Reply:
Makes absolutely sense. Good to go.